easybuild/easyconfigs/p/Perl/Perl-5.36.1-GCCcore-13.1.0.eb

```diff
@@ -0,0 +1,25 @@
+name = 'Perl'
+version = '5.36.1'
+
+homepage = 'https://www.perl.org/'
+description = """Larry Wall's Practical Extraction and Report Language
+
+This is a minimal build without any modules. Should only be used for build dependencies.
```

Comment:
Forgot about the description, if we go for this approach and skip the minimal variant altogether going forward, what should we write here?

Reply:
maybe drop the ` Should only be used for build dependencies.`, and just leave it at `This is a minimal build without any modules. `